modules/gitops-creating-and-configuring-user-defined-appproject-instance-with-target-namespaces.adoc

```diff
@@ -0,0 +1,75 @@
+====
+
+.Prerequisites
+* You are logged in to the {OCP} cluster as an administrator.
+* You have installed {gitops-title} 1.13.0  or a later version on your {OCP} cluster.
+
+.Procedure
+
+. Create and configure a user-defined `AppProject` instance in the `openshift-gitops` control plane namespace to specify the target namespaces in the `.spec.sourceNamespaces` field:
+.. From the *Project* drop-down menu, select the `openshift-gitops` project.
+.. In the *Administrator* perspective of the web console, click *Operators* -> *Installed Operators* -> *{gitops-title}* and go to the *AppProject* tab.
+.. Click *Create AppProject* and enter the following configuration in the YAML view:
++
+.Example user-defined `AppProject` instance
+[source,yaml]
+----
+kind: AppProject
+apiVersion: argoproj.io/v1alpha1
+metadata:
+  name: project-one # <1>
+  namespace: openshift-gitops # <2>
+spec:
+  sourceNamespaces: # <3>
+  - dev # <4>
+  - app-team-* # <5>
+----
+<1> The name of the user-defined `AppProject` instance.
+<2> The control plane namespace where you want to run the user-defined `AppProject` instance.
+<3> List of non-control plane namespaces for creating and managing `Application` resources.
+<4> The name of the target namespace for the Argo CD server to create and manage `Application` resources.
+<5> With wildcards (`pass:[*]`), specifies the name of the target namespaces matching the pattern `app-team-pass:[*]`, such as `app-team-1`, `app-team-2`, and so on for the Argo CD server to create and manage `Application` resources.
+.. Click *Create*.
++
+The *AppProjects* page displays the created user-defined `AppProject` instance. 
++
+[NOTE]
+====
+When a target namespace is specified under the `sourceNamespaces` field, the Operator adds the `argocd.argoproj.io/managed-by-cluster-argocd` label to the specified namespace. 
+
+.Example `dev` target namespace
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: dev
+  labels:
+    argocd.argoproj.io/managed-by-cluster-argocd: spring-petclinic # <1>
+    kubernetes.io/metadata.name: dev # <2>
+----
+<1> The namespace of the user-defined cluster-scoped Argo CD instance.
+<2> The target namespace for the Argo CD server to create and manage `Application` resources.
+====
+
+. Verify that Operator adds the `argocd.argoproj.io/managed-by-cluster-argocd` label to the specified namespace:
+.. Go to *Administration* -> *Namespaces* and click *Create Namespace*.
+.. In the *Create Namespace* dialog box, provide the *Name* and click *Create*.
++
+For example, to create `dev` target namespace, enter `dev` in the *Name* field. You can repeat the previous sub-steps of step 2 to create the `app-team-1` and `app-team-2` target namespaces.
```

Comment:
```suggestion
For example, to create `dev` target namespace, enter `dev` in the *Name* field. You can repeat the previous steps to create the `app-team-1` and `app-team-2` target namespaces.
```
best not to directly specify step numbers in case something is added later and renumbering happens. I think just saying "previous steps" would be OK.